nodeJSTypeScript/Dockerfile

```diff
@@ -1,41 +1,53 @@
 # ARGUMENTS --------------------------------------------------------------------
 ##
-# Board architecture
+# Base cross toolchain sdk container version
 ##
-ARG IMAGE_ARCH=
-
-
-
+ARG CROSS_SDK_BASE_TAG=3.1.0-bookworm
 ##
 # Base container version
 ##
 ARG BASE_VERSION=3.1.0-bookworm
+##
+# Board architecture
+# arm or arm64
+##
+ARG IMAGE_ARCH=
 
 ##
-# Application Name
+# Directory of the application inside container
 ##
-ARG APP_EXECUTABLE=__change__
```

Comment:
Please revert it, although this is fast, NPM packages generally need to compile stuff for foreign architectures